src/ipmanager/ui/templates/ui/single_group_view.html

```diff
@@ -0,0 +1,76 @@
+{% extends 'ui/base.html' %}
+{% block content %}
+<h1> {{group.key}} </h1>
+
+<h4 style="margin-bottom: 0.5rem;">Group Information</h4>
+<div style="width: 80vw; background-color: rgb(196, 203, 201);">
```

Comment:
For now, I would suggest leaving off any inline style attributes. In my experience, it is better to do the styling as a second pass over the application after all the views and forms are in place. Having everything in place sometimes exposes patterns that are harder to see in the isolation of a single view.